p2p/transport/quicreuse/connmgr.go

```diff
@@ -104,22 +104,22 @@ func (c *ConnManager) ListenQUIC(addr ma.Multiaddr, tlsConf *tls.Config, allowWi
 		return nil, err
 	}
 
-	c.connsMu.Lock()
-	defer c.connsMu.Unlock()
+	c.quicListenersMu.Lock()
+	defer c.quicListenersMu.Unlock()
 
 	key := laddr.String()
-	entry, ok := c.conns[key]
+	entry, ok := c.quicListeners[key]
 	if !ok {
-		conn, err := c.transportForListen(netw, laddr)
+		tr, err := c.transportForListen(netw, laddr)
 		if err != nil {
 			return nil, err
 		}
-		ln, err := newConnListener(conn, c.serverConfig, c.enableDraft29)
+		ln, err := newQuicListener(tr, c.serverConfig, c.enableDraft29)
```

Comment:
can you rename this function's argument at [definition](https://github.com/libp2p/go-libp2p/blob/7773888b80bdab3c1e6292941898cd3348122575/p2p/transport/quicreuse/listener.go#L41) to tr? 
```
func newQuicListener(c refCountedQuicTransport, quicConfig *quic.Config, enableDraft29 bool) (*quicListener, error) {
```